Notebooks/DC2 Coadd Run1.1p GCR access -- Tutorial (part1).ipynb

```diff
@@ -0,0 +1,1401 @@
+    "#######\n",
+    "\n",
+    "# Here is a template to fill for g1\n",
+    "catalog.add_modifier_on_derived_quantities('g1',\n",
+    "                                           #### This would be removed in the tutorial\n",
+    "                                           g1_modif, \n",
+    "                                           'Ixx', 'Iyy', 'Ixy'\n",
+    "                                           ########\n",
+    "                                          )\n",
+    "# Define in the same way g2, sigma, psf_g1, psf_g2 psf_sigma\n",
+    "\n",
+    "### The following would be hidden\n",
+    "catalog.add_modifier_on_derived_quantities('g2', g2_modif, 'Ixx', 'Iyy', 'Ixy')\n",
+    "catalog.add_modifier_on_derived_quantities('sigma', sigma_modif, 'Ixx', 'Iyy', 'Ixy')\n",
+    "\n",
+    "catalog.add_modifier_on_derived_quantities('psf_g1', g1_modif, 'IxxPSF', 'IyyPSF', 'IxyPSF')\n",
+    "catalog.add_modifier_on_derived_quantities('psf_g2', g2_modif, 'IxxPSF', 'IyyPSF', 'IxyPSF')\n",
+    "catalog.add_modifier_on_derived_quantities('psf_sigma', sigma_modif, 'IxxPSF', 'IyyPSF', 'IxyPSF')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 3: Extract sample\n",
+    "\n",
+    "Now that we have all the pieces, let's extract the quantities specified below from the catalog:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 99,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "quantities = ['ra', 'dec', \n",
+    "              'mag_i', 'i_SNR', 'psf_fwhm_i',\n",
+    "              'g1', 'g2', 'sigma',\n",
+    "              'psf_g1', 'psf_g2', 'psf_sigma']\n",
+    "\n",
+    "# Would be hidden\n",
+    "data = catalog.get_quantities(quantities, \n",
+    "                              native_filters=[(lambda x: x==4850, 'tract')],\n",
+    "                              filters=filters)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 4: Size and ellipticity residuals as a function of magnitude and seeing\n",
+    "\n",
+    "Try to reproduce plots similar to the ones you can find in the PSF section of your favorite experiment's shape catalog paper (for instance Section 4. of [Mandelbaum et al. 2017](https://arxiv.org/pdf/1705.06745.pdf)).\n",
+    "\n",
+    "For instance you can look at the fractional difference in size, as a function of magnitude, or seeing. You can also look at the distribution of ellipticity residuals, make sure they are centered on 0 , and again see if you can spot a dependence on seeing or magnitude."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 100,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<matplotlib.text.Text at 0x2ae54ada36d8>"
+      ]
+     },
+     "execution_count": 100,
+     "metadata": {},
+     "output_type": "execute_result"
+    },
+    {
+     "data": {
```

Comment:
Please remove plots.  If you have an image you want to persist, make a PNG, and include in assets.